python/adelphi/tests/integration/test_cql.py

```diff
@@ -0,0 +1,146 @@
+import glob
+import logging
+import os
+import shutil
+import sys
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+if os.name == 'posix' and sys.version_info[0] < 3:
+    import subprocess32 as subprocess
+else:
+    import subprocess
+
+from tests.integration import SchemaTestCase, setupSchema, dropKeyspace
+from tests.util.schemadiff import cqlDigestGenerator
+from tests.util.schema_util import get_schema
+
+log = logging.getLogger('adelphi')
+
+CQL_REFERENCE_SCHEMA_PATH = "tests/integration/resources/cql-schemas/{}.cql"
+CQL_REFERENCE_KS0_SCHEMA_PATH = "tests/integration/resources/cql-schemas/{}-ks0.cql"
+
+def digestSet(schemaFile):
+    rv = set()
+    for (_, digest) in cqlDigestGenerator(schemaFile):
+        rv.add(digest)
+    return rv
+
+
+def logCqlDigest(schemaFile, digestSet):
+    for (cql, digest) in cqlDigestGenerator(schemaFile):
+        if digest in digestSet:
+            log.info("Digest: {}, CQL: {}".format(digest,cql))
+
+
+class TestCql(SchemaTestCase):
+
+    # ========================== Unittest infrastructure ==========================
+    def setUp(self):
+        super(TestCql, self).setUp()
+        log.warning("Creating schema")
+        setupSchema(self.buildSchema())
+
+
+    def tearDown(self):
+        super(TestCql, self).tearDown()
+        log.warning("Dropping keyspace testkeyspace my_ks_0")
+        dropKeyspace("my_ks_0")
+        log.warning("Dropping keyspace testkeyspace my_ks_1")
+        dropKeyspace("my_ks_1")
```

Comment:
In this case test schemas are managed via instance methods but it's entirely up to the test author how they want to do this.